pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -438,3 +438,21 @@ def test_timestamp_multiindex_indexer():
     )
     should_be = pd.Series(data=np.arange(24, len(qidx) + 24), index=qidx, name="foo")
     tm.assert_series_equal(result, should_be)
+
+
+def test_multiindex_loc_order():
+    # GH 22797
+    # Try to respect order of keys given for MultiIndex.loc
+    df = pd.DataFrame(
+        np.arange(12).reshape((4, 3)),
+        index=[["a", "a", "b", "b"], [1, 2, 1, 2]],
+        columns=[["Ohio", "Ohio", "Colorado"], ["Green", "Red", "Green"]],
+    )
+
+    res = df.loc[["b", "a"], :]
```

Comment:
can you also do 'a', 'b' on both of these cases

Reply:
Done. I also have a additionnal question about a possible test case where I am not sure of the expected output.